lib/get_parser.py

```diff
@@ -44,10 +46,20 @@ def __call__(
         setattr(namespace, self.dest, values)
 
 
-def get_base_parser() -> argparse.ArgumentParser:
+def get_base_parser(
```

Comment:
I don't love the idea of adding arguments to the `get_base_parser` function as needed, just because that could turn into a mess if we extract it out to many kinds of parsers.  What would you think about having the base parser stay the same, that is, the common set of flags that every binary needs, and then having different module-level functions in this file that the binaries could call?  So something like:

```python
def get_base_parser():
    same_args_as_now()
    return parser

def get_jobid_parser():
    base_parser = get_base_parser()
    parser.add_argument("-J", "--jobid"....)
    return parser
```

and so on?

Reply:
Okay.  That sounds reasonable.  I've pushed that change.